src/Installers/Windows/SharedFrameworkBundle/1036/thm.wxl

```diff
@@ -0,0 +1,61 @@
+  <String Id="HelpCloseButton">&amp;Fermer</String>
+  <String Id="InstallLicenseLinkText">[WixBundleName] &lt;a href="#"&gt;termes du contrat de licence&lt;/a&gt;.</String>
+  <String Id="InstallAcceptCheckbox">J'&amp;accepte les conditions générales de la licence</String>
+  <String Id="InstallOptionsButton">&amp;Options</String>
+  <String Id="InstallInstallButton">&amp;Installer</String>
+  <String Id="InstallCloseButton">&amp;Fermer</String>
+  <String Id="OptionsHeader">Options d'installation</String>
+  <String Id="OptionsLocationLabel">Emplacement de l'installation :</String>
+  <String Id="OptionsBrowseButton">&amp;Parcourir</String>
+  <String Id="OptionsOkButton">&amp;OK</String>
+  <String Id="OptionsCancelButton">&amp;Annuler</String>
+  <String Id="ProgressHeader">Progression de l'installation</String>
+  <String Id="ProgressLabel">En cours :</String>
+  <String Id="OverallProgressPackageText">Initialisation...</String>
+  <String Id="ProgressCancelButton">&amp;Annuler</String>
+  <String Id="ModifyHeader">Modifier l'installation</String>
+  <String Id="ModifyRepairButton">&amp;Réparer</String>
+  <String Id="ModifyUninstallButton">&amp;Désinstaller</String>
+  <String Id="ModifyCloseButton">&amp;Fermer</String>
+  <String Id="SuccessRepairHeader">Réparation terminée avec succès</String>
+  <String Id="SuccessUninstallHeader">Désinstallation terminée avec succès</String>
+  <String Id="SuccessInstallHeader">Installation terminée avec succès</String> 
+  <String Id="SuccessHeader">Installation/désinstallation réussie</String>	
+  <String Id="SuccessLaunchButton">&amp;Démarrer</String>
+  <String Id="SuccessRestartText">Vous devez redémarrer votre ordinateur avant de pouvoir utiliser le logiciel.</String>
+  <String Id="SuccessRestartButton">&amp;Redémarrer</String>
+  <String Id="SuccessCloseButton">&amp;Fermer</String>
+  <String Id="FailureHeader">Échec de l'installation</String>
+  <String Id="FailureInstallHeader">Échec de l'installation</String>
+  <String Id="FailureUninstallHeader">Échec de la désinstallation</String>
+  <String Id="FailureRepairHeader">Échec de la réparation</String> 
+  <String Id="FailureHyperlinkLogText">Un ou plusieurs problèmes sont à l'origine de l'échec de l'installation. Corrigez ces problèmes, puis recommencez l'installation. Pour plus d'informations, voir le &lt;a href="#"&gt;fichier journal&lt;/a&gt;.</String>
+  <String Id="FailureRestartText">Vous devez redémarrer votre ordinateur pour terminer l'opération de restauration du logiciel.</String>
+  <String Id="FailureRestartButton">&amp;Redémarrer</String>
+  <String Id="FailureCloseButton">&amp;Fermer</String>
+  <String Id="FilesInUseHeader">Fichiers en cours d'utilisation</String>
+  <String Id="FilesInUseLabel">Les applications suivantes utilisent des fichiers nécessitant une mise à jour :</String>
+  <String Id="FilesInUseCloseRadioButton">&amp;Fermer les applications essayez de les ouvrir de nouveau.</String>
```

Comment:
FYI, this entry - that also appears in the server bundle installer - is not correct. The correct form would be:

```
Fermer les applications et essayer de les ouvrir de nouveau.
```

That said, it's a bit hard to read. Here's a much more natural version:

```
Fermer les applications et tenter de les redémarrer.
```

Reply:
Thank you for pointing this out. I take it you were referring to the "z"/"r" usage being inconsistent in the original translation which should have been Fermez instead?

I've updated the PR with your suggestion.

Reply:
@joeloff thanks!

> I take it you were referring to the "z"/"r" usage being inconsistent in the original translation which should have been Fermez instead?

Actually, inconsistency wasn't the only issue. In French, a verb ending with `ez` without a personal pronoun (`vous` in this case) always indicates that a sentence uses the imperative mood with a second-person subject.

In this case, using the imperative mood would be - at beast - misleading, as it would imply that users have to close and restart the listed apps themselves before continuing (which would actually lead to the same error when trying to re-execute the installer, since the files would still be locked).

Here, the user is given a choice - allowing the installer to temporarily close apps holding locks on files it must update or restarting the computer - so an infinitive sentence is a much better option.